### PR TITLE
Fix intersect_percentage API

### PR DESCRIPTION
**Pull request by ericboucher**
- Fix the intersect_percentage calculation by considering nodata pixels as valid for the denominator.
- Add intersect_area statistic

### CODE REVIEW

File: api/app/main.py [hunk 1]
```diff
@@ -356,7 +356,7 @@ def stats_demo(
     features = _calculate_stats(
         zones_filepath,
         geotiff,
-        stats=" ".join(["min", "max", "mean", "median", "sum", "std"]),
+        stats=" ".join(DEFAULT_STATS),
```
Comment (laurentS) [inline api/app/main.py:359]: You've dropped "sum" and "std" in doing this. did you mean to?

File: api/app/zonal_stats.py [hunk 2]
```diff
@@ -335,19 +334,24 @@ def calculate_stats(
...
             total = masked.count()
-            # Avoid dividing by 0
             if total == 0:
-                return 0
-            percentage = (intersect_operator(masked, intersect_baseline)).sum()
-            return percentage / total
+                return 0.0
+            return float((intersect_operator(masked, intersect_baseline)).sum())
+
+        def intersect_area(masked) -> float:
```
Comment (laurentS) [inline api/app/zonal_stats.py:348]: You have a lot of `float` values which are not of the same type. You would probably make it easier to follow the code (and avoid bugs) by doing something like:
```python
AreaInSqKmPerPixel = NewType("AreaInSqKmPerPixel", float)

def intersect_area(masked) -> AreaInSqKmPerPixel:
    ...
    return AreaInSqKmPerPixel(myvalue)
```

File: api/app/main.py (hunk 1 above)
Comment (ericboucher) [reply api/app/main.py:359]: nice catch! fixed

File: api/app/zonal_stats.py (hunk 2 above)
Comment (ericboucher) [reply api/app/zonal_stats.py:348]: it's just SqKm, but I see your point. Adding SqKm and Percentage

**Pull request merged**